### ISSUE TITLE
[Help Wanted] Update & Redesign

### ISSUE THREAD

**Issue by fedebotu** [ai4co/ai4co-homepage#1]
We would like to give the AI4CO homepage updates and a fresh look!

## Content
- [ ] **General content**: update content to feel more like a welcoming open-source community (e.g., "People" "Who are we", "Open Source"); at the moment IMHO feels a bit like a company 
- [ ] **Research page**: add a list of all papers under the AI4CO Github and spotlight updates such as acceptance to conferences
- [ ] **Blog Posts**: we should allow people to make blog posts (see below), which would be a neat feature, in particular for hosting project pages

## Redesign
- [ ] **Modularization**: so that we don't have to add hardcoded HTML code (e.g. modifying template websites)

---

Your help is needed: please drop in ideas!

**Comment by fedebotu** [comment]
Closing as stale for now, we shall do for KDD @cbhua ~

**Comment by cbhua** [comment]
Yeah, I will update the home page for KDD and the workshop these days. It has been quite a long time since this site was last updated.
